CODEOWNERS

```diff
@@ -0,0 +1,40 @@
+# This is a comment.
+# Each line is a file pattern followed by one or more owners.
+
+# These owners will be the default owners for everything in
+# the repo. Unless a later match takes precedence,
+# @global-owner1 and @global-owner2 will be requested for
+# review when someone opens a pull request.
+modules/cilium @qvistgaard
+modules/loki @qvistgaard
+modules/prometheus @qvistgaard
+
+
+# Order is important; the last matching pattern takes the most
```

Comment:
These are not relevant for this repository. I can remove them if you want?

Reply:
sure go ahead